server/files/api_views.py

```diff
@@ -21,14 +22,24 @@ def perform_destroy(self, instance):
 
     def create(self, request):
         (metadata, file) = (json.loads(self.request.data["metadata"]), self.request.data["file"])
```

Comment:
No need for the extra value unpacking here, easier to spot diffs later on.
```suggestion
        metadata = json.loads(self.request.data["metadata"])
        file =  self.request.data["file"]
```